google_photos_takeout_helper/__main__.py

```diff
@@ -744,9 +744,9 @@ def _walk_with_tqdm(res, bar: _tqdm):
             logger.info(f" - you have full list in {f.name}")
 
     logger.info('')
-    logger.info('Sooo... what now? You can see README.md for what nice G Photos alternatives I found and recommend')
+    logger.info('Sooo... what now? You can see README.md for what nice G Photos alternatives I have found and would recommend')
```

Comment:
This no

Reply:
Could you please give a reason?

Reply:
Because I **strongly recommend them** because they are _**awesome**_

I shove them into your mouth because they are *so good*

I attatch them directly to the repo because they are *so good*

I praise their creators and pipe my donations from this repo directly to them because they are *so good*